orca-webhook/src/test/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTaskSpec.groovy

```diff
@@ -280,6 +280,39 @@ class CreateWebhookTaskSpec extends Specification {
     stage.context.statusEndpoint == null
   }
 
+  // Tests https://github.com/spinnaker/spinnaker/issues/2163
```

Comment:
TBH not sure how often we link out to a specific issue in code ... not against it but we should likely also (or instead?) have a `fixes spinnaker/spinnaker#2163`

Reply:
This bug was subtle enough I feel a link to the issue provides context on what the test is trying to do

> should have a fixes spinnaker/spinnaker#2163

As in a commit comment? yah will do